src/Init/Init_FFTW.cpp

```diff
@@ -40,8 +51,19 @@ size_t ComputePaddedTotalSize(int* size) {
 //
 // Return      :  length of array that is large enough to store FFT input and output
 //-------------------------------------------------------------------------------------------------------
+//int ComputeTotalSize(int* size) {
+//   return size[0]*size[1]*size[2];
+//} // FUNCTION : ComputeTotalSize
 size_t ComputeTotalSize(int* size) {
```

Comment:
Since when using `MPI` the returned value here is the size required by each rank instead of the total size, so maybe we can rename it to something like `ComputeRequiredSize` to avoid misunderstanding~?